src/main/python/apache/aurora/executor/bin/thermos_executor_main.py

```diff
@@ -183,6 +183,14 @@
     help='Maximum number of rotated stdout/stderr logs emitted by the thermos runner.')
 
 
+app.add_option(
```

Comment:
This option should already exist by default:
```
vagrant@aurora:~$ aurora/dist/thermos_executor.pex --long-help
    --log_to_disk=[scheme:]LEVEL
                        The level at which logging to disk [default: INFO].
                        Takes either LEVEL or scheme:LEVEL, where LEVEL is one
                        of ['INFO', 'NONE', 'WARN', 'ERROR', 'DEBUG', 'FATAL']
                        and scheme is one of ['google', 'plain'].
```
So throwing in the following into your scheduler options should already to the trick without the need for code modifications:

    -thermos_executor_flags="--log_to_disk=google:INFO ....."


This fixes the __main__.log to just contain INFO and above. The  `thermos_runner.log` still contains debug message though. This needs to be addressed elsewhere though.